### ISSUE TITLE
Change CONTAINER to <container_id>

### ISSUE THREAD

**Issue by owencyr** [docker/docker.github.io#10017]
File: [engine/reference/commandline/cp.md](https://docs.docker.com/engine/reference/commandline/cp/)

As a first time user of docker,
it was hard to parse from the docker cp page alone that I was meant to docker cp using the container id and not the tag of the container I had just built.

**Comment by craig-osterhout** [comment]
Thanks for sharing the issue. The content is maintained in an upstream repo. If the issue is still applicable, please help create an issue in the source repo at https://github.com/docker/cli/tree/master/docs/reference/commandline
